UI/Views/UpdaterPage.xaml

```diff
@@ -150,10 +150,11 @@
 
```

Comment:
Pls make sure there are no errors or warnings due to editor config in any of the checked-in files